packages/@webex/plugin-meetings/src/config.ts

```diff
@@ -86,8 +86,8 @@ export default {
     installedOrgID: undefined,
     experimental: {
       enableMediaNegotiatedEvent: false,
-      enableUnifiedMeetings: false,
-      enableAdhocMeetings: false,
+      enableUnifiedMeetings: true,
```

Comment:
Made these values to true `enableAdhocMeetings` & `enableAdhocMeetings` by default as of beta.
@arun3528

Reply:
@arun3528 @Neeraj-swarnkar @marcin-bazyl 

Should we also consider moving this out of experimental now that we are enabling it by default?

Reply:
lets keep it as the USM migration will take time so if someone has issues in v3 they can turn it off too . we can remove it in later release

Reply:
Thanks for update @arun3528